.github/workflows/darwin.yaml

```diff
@@ -99,9 +99,6 @@ jobs:
             - name: Build example All Clusters Server
               run: |
                   scripts/examples/gn_build_example.sh examples/all-clusters-app/linux out/debug/all-clusters-app chip_config_network_layer_ble=false
-            - name: Build example OTA Provider
-              run: |
-                  scripts/examples/gn_build_example.sh examples/ota-provider-app/linux out/debug/ota-provider-app chip_config_network_layer_ble=false
             - name: Build example OTA Requestor
```

Comment:
do you want to remove the requestor too?

Reply:
It's used in the OTA tests, with Matter.framework acting as the provider.